src/main/kotlin/com/mparticle/kits/ApptentiveKitUtils.kt

```diff
@@ -12,18 +16,27 @@ import com.mparticle.MParticle
 object ApptentiveKitUtils {
     @JvmStatic
     fun registerApptentiveActivityContext(callback: ApptentiveActivityInfo) {
+        val broadcastReceiver = object : BroadcastReceiver() {
+            override fun onReceive(context: Context, intent: Intent) {
+                if (intent.action?.startsWith(MParticle.ServiceProviders.BROADCAST_ACTIVE) == true) {
+                    Apptentive.registerApptentiveActivityInfoCallback(callback)
+                }
+            }
+        }
         if (MParticle.getInstance()?.isKitActive(MParticle.ServiceProviders.APPTENTIVE) == true) {
             Apptentive.registerApptentiveActivityInfoCallback(callback)
+            Log.d("ApptentiveKitUtils", "registerApptentiveActivityContext: kit is active")
         } else {
-            val filter =
-                IntentFilter(MParticle.ServiceProviders.BROADCAST_ACTIVE + MParticle.ServiceProviders.APPTENTIVE);
-            callback.getApptentiveActivityInfo()?.registerReceiver(object : BroadcastReceiver() {
-                override fun onReceive(context: Context, intent: Intent) {
-                    if (intent.action?.startsWith(MParticle.ServiceProviders.BROADCAST_ACTIVE) == true) {
-                        Apptentive.registerApptentiveActivityInfoCallback(callback)
-                    }
-                }
-            }, filter)
+            val filter = IntentFilter(MParticle.ServiceProviders.BROADCAST_ACTIVE + MParticle.ServiceProviders.APPTENTIVE)
+            if (Build.VERSION.SDK_INT > Build.VERSION_CODES.TIRAMISU) {
+                callback.getApptentiveActivityInfo()
+                    ?.registerReceiver(broadcastReceiver, filter, RECEIVER_EXPORTED)
+                Log.d("ApptentiveKitUtils", "registerApptentiveActivityContext: kit is active SDK 33+")
```

Comment:
```suggestion
                Logger.debug("ApptentiveKitUtils", "registerApptentiveActivityContext: kit is active SDK 33+")
```

can you please use here Logger.debug

Reply:
Updated.